src/memory/client/MemoryBrowser.tsx

```diff
@@ -77,21 +77,52 @@ interface State {
   bytesPerRow: number;
   bytesPerGroup: number;
   endianness: 'le' | 'be';
+  childNames: { id: string; name: string }[];
 }
 
 export class MemoryBrowser extends React.Component<Props, State> {
   private addressReq = '';
   private lengthReq = '512';
+  private childReq = '0';
```

Comment:
Please check the types here. As mentioned in the cdt-amalgamator change indexes should be numbers.

Reply:
I have updated it